utils/spdx/src/main/kotlin/Utils.kt

```diff
@@ -164,3 +187,6 @@ private fun getLicenseTextFile(id: String, dir: File): File? =
             dir.resolve(filename).takeIf { it.isFile }
         }.firstOrNull()
     }
+
+private fun addScanCodeLicenseTextsDir(licenseTextDirectories: List<File>): List<File> =
```

Comment:
IMO the need for this helper function is not nice. Could we get rid of it by changing `getLicenseTextReader()` to take varargs instead of a list of files for the directories?

Reply:
This would change the signature and we would still have to add the scancode dir in `hasLicenseText` and `getLicenseTextReader`. Would it be ok to improve that outside of this PR?

Reply:
Yes, we can do that later. Maybe add a `TODO` to this function to remember?